distributed/worker.py

```diff
@@ -283,6 +280,15 @@ def heartbeat(self):
         else:
             logger.debug("Heartbeat skipped: channel busy")
 
+    def get_metrics(self):
+        core = dict(executing=len(self.executing),
+                    in_memory=len(self.data),
+                    ready=len(self.ready),
+                    in_flight=len(self.in_flight_tasks))
+        custom = {k: metric(self) for k, metric in self.metrics.items()}
+
+        return merge(core, custom, self.monitor.recent())
```

Comment:
The last one wins in `merge` right? So should it not be `merge(custom, self.monitor.recent(), core)`? Maybe we need to add `in_flight` to the test (I think the test uses `cpu` at the moment) so that we check that a metric in `custom` does not override anything in `core`.

Reply:
Good point